README.md

```diff
@@ -104,13 +104,23 @@ The configuration for queues/topics can be changed or reset to default on **Demo
 | Inventory service request queue   | `inventory.request`                                      |
 | Inventory service response queue  | `inventory.response`                                     |
 
+Part of the configuration can be changed in **application.properties** file.
```

Comment:
I'm thinking maybe we can add another column in the table to display either the property name in "application.properties" or display "N.A." if not applicable.
And at the beginning of the readme, mention "application.properties" file.
So let's remove this line here and handle the case in [PDA-1451](https://jira.parasoft.com/browse/PDA-1451)